src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderCard.tsx

```diff
@@ -101,22 +105,41 @@ export const LimitOrderCard: FC<LimitOrderCardProps> = ({
     onCancelClick?.(uid)
   }, [onCancelClick, uid])
 
-  const formattedPercentage = (filledDecimalPercentage * 100).toFixed(2)
-
   const sellAmountCryptoPrecision = useMemo(
-    () => fromBaseUnit(sellAmountCryptoBaseUnit, sellAsset?.precision ?? 0),
+    () => fromBaseUnit(sellAmountCryptoBaseUnit, sellAsset?.precision ?? 0, 6),
     [sellAmountCryptoBaseUnit, sellAsset?.precision],
   )
 
   const buyAmountCryptoPrecision = useMemo(
-    () => fromBaseUnit(buyAmountCryptoBaseUnit, buyAsset?.precision ?? 0),
+    () => fromBaseUnit(buyAmountCryptoBaseUnit, buyAsset?.precision ?? 0, 6),
     [buyAmountCryptoBaseUnit, buyAsset?.precision],
   )
 
+  const executedBuyAmountCryptoPrecision = useMemo(
+    () =>
+      executedBuyAmountCryptoBaseUnit
+        ? fromBaseUnit(executedBuyAmountCryptoBaseUnit, buyAsset?.precision ?? 0, 6)
+        : '0',
+    [executedBuyAmountCryptoBaseUnit, buyAsset?.precision],
+  )
+
+  const executedSellAmountCryptoPrecision = useMemo(
+    () =>
+      executedSellAmountCryptoBaseUnit
+        ? fromBaseUnit(executedSellAmountCryptoBaseUnit, sellAsset?.precision ?? 0, 6)
+        : '0',
+    [executedSellAmountCryptoBaseUnit, sellAsset?.precision],
+  )
+
   const limitPrice = useMemo(() => {
-    return bn(buyAmountCryptoPrecision).div(sellAmountCryptoPrecision).toFixed()
+    return bn(buyAmountCryptoPrecision).div(sellAmountCryptoPrecision).toFixed(6)
```

Comment:
Ditto precision and notifying users with ~

Reply:
Done in [8eeca7b](https://github.com/shapeshift/web/pull/8993/commits/8eeca7b14798c70145af9f3b9cd5a76d8dc19695)